pydruid/utils/filters.py

```diff
@@ -31,18 +31,19 @@ def __init__(self, **args):
             self.filter = {"filter": {"type": "javascript",
                                       "dimension": args["dimension"],
                                       "function": args["function"]}}
-
         elif args["type"] == "and":
             self.filter = {"filter": {"type": "and",
                                       "fields": args["fields"]}}
-
         elif args["type"] == "or":
             self.filter = {"filter": {"type": "or",
                                       "fields": args["fields"]}}
-
         elif args["type"] == "not":
             self.filter = {"filter": {"type": "not",
                                       "field": args["field"]}}
+        elif args["type"] == "in":
+            self.filter = {"filter": {"type": "in",
+                                      "dimension": args["dimension"],
+                                      "values": args["values"]}}
```

Comment:
minor nit : InFilter also supports extractionfn, would be nice to add that too. I'm fine with doing that in a separate PR too and not blocking this one on it.